lib/coverband-service-client.rb

```diff
@@ -22,7 +24,7 @@ def self.service_disabled_dev_test_env?
     def self.report_coverage
       # for now disable coverband reporting in test & dev env by default
       if Coverband.configuration.verbose
-        puts "Coverband: disabled for #{COVERBAND_ENV}, set COVERBAND_ENABLE_DEV_MODE or COVERBAND_ENABLE_TEST_MODE to enable"
+        puts "Coverband: disabled for #{COVERBAND_ENV}, set COVERBAND_ENABLE_DEV_MODE or COVERBAND_ENABLE_TEST_MODE to enable" if Coverband.configuration.verbose || COVERBAND_ENABLE_DEV_MODE
```

Comment:
I guess we need to use puts here since logger is not available yet?

Reply:
correct